Configure

```diff
@@ -2714,6 +2719,19 @@ exit(0);
 # Helpers and utility functions
 #
 
+# Death handler, to print a helpful message in case of failure #######
+#
+sub death_handler {
+    my $build_file = $target{build_file} // "build file";
+    print STDERR <<"_____";
+
+Failure!  $build_file wasn't produced
```

Comment:
I'd say that "was not" would sound so much better here. I for one also miss stop after "produced"

Reply:
I disagree with you about "wan't"
I agree with you about the period (fixed)